docs/next/guides/accessories-and-menus/keyboard-shortcuts.md

```diff
@@ -73,20 +75,24 @@ By default, Handsontable features the following keyboard shortcuts:
-| <kbd>Ctrl</kbd> + <kbd>V</kbd><br>or<br><kbd>Shift</kbd> + <kbd>Insert</kbd> | <kbd>Cmd</kbd> + <kbd>V</kbd>                   | Paste into the current cell                                                                           |
-| <kbd>Ctrl</kbd> + <kbd>Enter</kbd>                                           | <kbd>Cmd</kbd> + <kbd>Enter</kbd>               | When editing:<br>fill all currently-selected cells with the contents of the currently-edited cell     |
-| <kbd>Ctrl</kbd> + <kbd>Z</kbd>                                               | <kbd>Cmd</kbd> + <kbd>Z</kbd>                   | Undo                                                                                                  |
-| <kbd>Ctrl</kbd> + <kbd>Y</kbd>                                               | <kbd>Cmd</kbd> + <kbd>Y</kbd>                   | Redo                                                                                                  |
+| Windows                                                                      | macOS                                             | Action                                                                                                                                                                                                                                           |
+| ---------------------------------------------------------------------------- | ------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| <kbd>Enter</kbd>                                                             | <kbd>Enter</kbd>                                  | Edit the current cell<br><br>When editing:<br>stop editing, save your changes, and move one cell down                                                                                                                                            |
+| <kbd>Shift</kbd> + <kbd>Enter</kbd>                                          | <kbd>Shift</kbd> + <kbd>Enter</kbd>               | Edit the current cell<br><br>When editing:<br>stop editing, save your changes, and move one cell up                                                                                                                                              |
+| <kbd>Ctrl</kbd> + <kbd>Enter</kbd>                                           | <kbd>Ctrl</kbd>/<kbd>Cmd</kbd> + <kbd>Enter</kbd> | Edit the current cell<br><br>When editing, with a single cell selected:<br>add a new line inside the cell<br><br>When editing, with multiple cells selected:<br>fill all currently-selected cells with the contents of the currently-edited cell |
```

Comment:
This is not multiple, but a range selection. [Multiple is non-contiguous ranges](https://handsontable.com/docs/selection/#selecting-ranges).

Reply:
@aninde - right, thank you! Fixed: [0ff58f6](https://github.com/handsontable/handsontable/pull/9271/commits/0ff58f656d1b9221c1921f7c165f4d4fd62d82ac)